<launch>
	<node pkg="refresh_ros" type="test_bt" name="test_bt" output="screen" />
</launch>
